CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 
 # Changelog
 
+## Forthcoming
+
+MotoROS2 *Humble* is now built against `micro_ros_motoplus` version `20230724`.
+
+MotoROS2 *Foxy* and *Galactic* still build against `micro_ros_motoplus` version `20221102`.
```

Comment:
I'm wondering whether we should actually give the Foxy and Galactic versions a new release as well.

That release would not update any of the micro-ROS packages proper, but would include the updates to `motoros2_interfaces`.

Reply:
I'd suggest we do.

Reply:
Backports submitted:

- https://github.com/Yaskawa-Global/micro_ros_motoplus/pull/6
- https://github.com/Yaskawa-Global/micro_ros_motoplus/pull/7

Reply:
Done.